client/packages/common/src/plugins/api/usePluginGraphql.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  JsonValue,
+  useAuthContext,
+  useGql,
+  useMutation,
+  useQuery,
+} from '@openmsupply-client/common';
+import { getSdk } from './operations.generated';
+
+const useQueryMethod = <I extends JsonValue, O extends JsonValue>(
```

Comment:
Wanted to expose two methods usePluginGraphql and async version (mutation), that's the main reason for this abstraction